packages/angular/src/generators/setup-ssr/lib/update-project-config.ts

```diff
@@ -72,4 +74,18 @@ export function updateProjectConfig(tree: Tree, schema: Schema) {
   };
 
   updateProjectConfiguration(tree, schema.project, projectConfig);
+
+  const workspaceConfig = readWorkspaceConfiguration(tree);
+  if (
+    workspaceConfig.tasksRunnerOptions?.default &&
+    !workspaceConfig.tasksRunnerOptions.default.options.cacheableOperations[
+      'server'
+    ]
```

Comment:
cacheableOperations is an array.

```suggestion
    !workspaceConfig.tasksRunnerOptions.default.options.cacheableOperations.includes(
      'server'
    )
```